rskj-core/src/main/java/co/rsk/rpc/JsonRpcRequestValidatorInterceptor.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.rpc;
+
+import co.rsk.rpc.exception.JsonRpcRequestPayloadException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.googlecode.jsonrpc4j.JsonRpcInterceptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+public class JsonRpcRequestValidatorInterceptor implements JsonRpcInterceptor {
+    private static final Logger logger = LoggerFactory.getLogger(JsonRpcRequestValidatorInterceptor.class);
+
+    private final Integer maxBatchRequestsSize;
+
+    public JsonRpcRequestValidatorInterceptor(Integer maxBatchRequestsSize) {
+        this.maxBatchRequestsSize = maxBatchRequestsSize;
+    }
+
+    private void validateRequest(JsonNode node) {
+        if (node.isArray() && node.size() > this.maxBatchRequestsSize) {
+            String msg = String.format("Cannot dispatch batch requests. %s is the max number of supported batch requests", this.maxBatchRequestsSize);
+            logger.error(msg);
```

Comment:
```suggestion
            logger.warn(msg);
```
as this is not an error in the node (just invalid user input), do you think we could use WARN here?